src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -276,6 +274,22 @@ uct_ib_mlx5_devx_memh_base_address(const uct_ib_mlx5_devx_mem_t *memh)
     return memh->address;
 }
 
+static uint64_t uct_ib_mlx5_devx_memh_iova(const uct_ib_mlx5_devx_mem_t *memh,
+                                           uint32_t iova_offset)
+{
+    uint64_t address;
+
+#if HAVE_IBV_DM
+    if (memh->dm != NULL) {
+        /* Device memory memory key is zero based */
+        return (uint64_t)memh->address + iova_offset;
```

Comment:
Should we align to `memh->address` to `UCT_IB_MD_MAX_MR_SIZE`  in that case?

Reply:
No